modules/imgui/imguimodule.cpp

```diff
@@ -153,31 +153,45 @@ ImGUIModule::ImGUIModule()
     });
 
     global::callback::keyboard->emplace_back(
-        [&](Key key, KeyModifier mod, KeyAction action) -> bool {
+        [&](Key key, KeyModifier mod, KeyAction action, bool isGuiWindow) -> bool {
             ZoneScopedN("ImGUI")
 
+            if (!isGuiWindow) {
```

Comment:
->
```cpp
if (!isGuiWindow || !_isEnabled) {
    return false:
}
else {
    return keyCallback(key, mod, action);
}
```

Reply:
This might be a personal preference, but do we really need the else statement here? To me it feels implied because of the return statement in the first if

But again, personal preference and I'm fine with adding the else if we think that makes it clearer

Reply:
Yea, I think the else statement is totally optional;  the comment was more about moving the `_isEnabled` part into the first `if`.
Alternatively of course:
`return isGuiWindow && _isEnabled ? keyCallback(key, mod, action) : false;`